odoo-shippable/scripts/library.sh

```diff
@@ -58,3 +58,20 @@ createuser_custom(){
 psql_create_role(){
     su - postgres -c "psql -c  \"CREATE ROLE ${1} LOGIN PASSWORD '${2}' SUPERUSER INHERIT CREATEDB CREATEROLE;\""
 }
+
+service_postgres_without_sudo(){
+    USER="${1}"
+    VERSIONS=$(pg_lsclusters  | sed '1d' | awk '{print $1}' )
+    for version in $VERSIONS; do
+        pg_dropcluster --stop $version main
+    done
+    adduser ${USER} postgres
+    chown -R ${USER}:${USER} /var/run/postgresql
+    for version in $VERSIONS; do
+        pg_createcluster -u ${USER} -g postgres -s /var/run/postgresql --start-conf auto --start $version main
```

Comment:
@moylop260 Can not force use port 5432,Since this port was already used for the previous cluster
![por_error](https://cloud.githubusercontent.com/assets/16363083/23219480/91ec9060-f8f5-11e6-9a46-cc7b002f4b13.png)